crates/viewer/re_redap_browser/src/servers.rs

```diff
@@ -130,16 +131,23 @@ impl Server {
                 .ok();
         }))
         .column_blueprint(|desc| {
-            let name = default_display_name_for_column(desc);
-            let name = name
-                .strip_prefix("rerun_")
-                .unwrap_or(name.as_ref())
-                .replace('_', " ");
+            let mut name = default_display_name_for_column(desc);
+
+            if name.starts_with("rerun_") {
+                name = name
+                    .strip_prefix("rerun_")
+                    .unwrap_or(name.as_ref())
+                    .replace('_', " ");
+            }
 
             let default_visible = if desc.entity_path().is_some_and(|entity_path| {
                 entity_path.starts_with(&std::iter::once(EntityPathPart::properties()).collect())
             }) {
-                true
+                // Property column, just hide indicator components
+                //TODO(ab): remove this when we no longer have indicator components
```

Comment:
```suggestion
                //TODO(#8129): remove this when we no longer have indicator components
```

makes it easier to find